tests/PHPStan/Analyser/data/bug-8015.php

```diff
@@ -16,15 +16,15 @@ function extractParameters(array $items): array
 			$config['things'] = [];
 			assertType('array{}', $config['things']);
 			foreach ($item as $thing) {
-				assertType('array<int, string>', $config['things']);
+				assertType('list<string>', $config['things']);
```

Comment:
I went through the diff multiple times now and still have no clue why this would change. anybody any ideas?

Reply:
-> https://github.com/phpstan/phpstan-src/pull/1796